contents/handbook/engineering/incidents.md

```diff
@@ -6,7 +6,7 @@ showTitle: true
 
 Incidents are going to happen.
 
-## When to raise an incident
+## When to raise an incident
```

Comment:
this didn't get render properly on https://posthog.com/handbook/engineering/incidents